### PR TITLE
Use implicit ActorRefFactory in HttpClient

### PR DESCRIPTION
**Pull request by gbougeard**

### CODE REVIEW

File: src/main/scala/tv/teads/github/api/util/HttpClient.scala [hunk 1]
```diff
@@ -20,7 +20,7 @@ case class SuccessfulRequest[T](response: T, rawResponse: HttpResponse) extends
 case class FailedRequest[T](statusCode: StatusCode) extends ClientResponse[T]
 
 object HttpClient {
-  def apply(req: HttpRequest): HttpClient = HttpClient(req, Nil)
+  def apply(req: HttpRequest)(implicit refFactory: ActorRefFactory, ec: ExecutionContext): HttpClient = HttpClient(req, Nil)
```
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/util/HttpClient.scala]: useless `ExecutionContext`

File: src/main/scala/tv/teads/github/api/services/GithubService.scala [hunk 2]
```diff
@@ -22,7 +22,7 @@ trait GithubService extends Service with PayloadFormats {
     queryParams:      Map[String, String],
     useTestMediaType: Boolean             = false,
     paginated:        Boolean             = false
-  ) = {
+  )(implicit refFactory: ActorRefFactory, ec: ExecutionContext) = {
```
Comment (pdalpra) [inline src/main/scala/tv/teads/github/api/services/GithubService.scala]: useless `ExecutionContext`

**Comment by gbougeard** [comment]
Should I also remove the `ExecutionContext` from `GithubService` other methods? (and maybe everywhere in others services?)

**Comment by pdalpra** [comment]
yep

**Pull request merged**